login-script.php

```diff
@@ -4,33 +4,42 @@
     include("./classes/functions.php");
 
     $email = sanitize($_POST["email"]);
-    // $passwd = sanitize($_POST["password"]);
+    $passwd = sanitize($_POST["passwd"]);
+    // $pass = $sql = "SELECT * from `password` WHERE `passwd` = '$passwd'";
 
     $sql = "SELECT * from `password` WHERE `email` = '$email'";
+    // $passwdDB = "SELECT * from `password` WHERE `passwd` = '$passwd'";
+    
+    
     $result = mysqli_query($conn, $sql);
-
+    $record = mysqli_fetch_assoc($result);
+    $encpass = $record["passwd"];
     if (mysqli_num_rows($result) == 1){
-        $record = mysqli_fetch_assoc($result);
-        echo("Successful login");
-        $sql = "SELECT * FROM `klant` WHERE `email` = '$email'";
-        $query = mysqli_query($conn, $sql);
-        $data = mysqli_fetch_assoc($query);
-        $userrole = "klant";
```

Comment:
I believe we have multiple user roles, now every use gets assigned  the role 'klant'?

Reply:
This branch is behind commits so this is a mini-scale task for the next sprint.